src/consolelogger.c

```diff
@@ -15,7 +15,7 @@ static char* vprintf_alloc(const char* format, va_list va)
 {
     char* result;
     int neededSize = vsnprintf(NULL, 0, format, va);
-    if (neededSize < 0)
+    if (neededSize < 0 || (neededSize + 1) < 0)
```

Comment:
Is this a case of not wanting to include safe math in this file? (and in etwlogger_driver)

Reply:
yes